client/apps/settings/client/curations/display/components/canvas/canvas_images.jsx

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import { Col, Row } from 'react-styled-flexboxgrid'
-import ImageUpload from 'client/apps/edit/components/admin/components/image_upload.coffee'
+import ImageUpload from '../../../../../../edit/components/admin/components/image_upload.coffee'
 
```

Comment:
This require stopped working specifically in tests (was fine in the client) -- error message looked like the resolver was missing one directory of depth, maybe related to recent changes @damassi made to test running?

Reply:
Ah sorry! I updated the path in the component but apparently didn't update the test; I wonder why nothing failed? I need to add a `roots` config to [jest setup](https://github.com/artsy/prediction/blob/master/package.json#L37).